src/components/editor/editor-modals/max-length-warning-modal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Button, Modal } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { CommonModal } from '../../common/modals/common-modal'
+
+export interface MaxLengthWarningModalProps {
+  show: boolean
+  onHide: () => void
+  maxLength: number
+}
+
+export const MaxLengthWarningModal: React.FC<MaxLengthWarningModalProps> = ({ show, onHide, maxLength }) => {
+  useTranslation()
+
+  return (
+    <CommonModal show={show} onHide={onHide} titleI18nKey={'editor.error.limitReached.title'} closeButton={true}>
+      <Modal.Body className={'limit-warning'}>
+        <Trans i18nKey={'editor.error.limitReached.description'} values={{ maxLength }} />
+        <br /><br/>
+        <b><Trans i18nKey={'editor.error.limitReached.advice'}/></b>
```

Comment:
This should be accomplished by CSS?

Reply:
Use <strong>

Reply:
I also meant the `<br/>`s

Reply:
Well yes. br should be used for actual line breaks. For margin please use the margin classes.

Reply:
Done in aef6aac